README.rst

```diff
@@ -1,20 +1,254 @@
+It integrates modern theming approaches, including design tokens and CSS runtime configurations, to ensure a consistent
+look and feel across MFEs and the legacy Open edX experience.
 
-```bash
-pip install git+https://github.com/edunext/tutor-modern-theming
-```
+Features
+--------
 
-## Usage
+- **Automatic Installation of Modern Themes**: Fetches and configures theme assets during the build process.
+- **Supports `PARAGON_THEME_URLS` Configuration**: Ensures MFEs can dynamically load themes via Paragon.
+- **Plugin Slots Integration**: Allows custom slot-based theming components to be loaded into MFEs.
+- **Consistent Header and Footer**: Enables cross-platform theming alignment between MFEs and legacy pages.
+- **Custom CLI Commands**: Provides an `enable-legacy-theme` command to simplify theme deployment.
 
-```bash
-tutor plugins enable tutor-modern-theming
-```
+Installation
+------------
 
+To install `tutor-modern-theming`, follow these steps:
+
+.. code-block:: bash
+
+    pip install git+https://github.com/eduNEXT/tutor-modern-theming.git
+
+Then enable the plugin:
+
+.. code-block:: bash
+
+    tutor plugins enable tutor-modern-theming
+
+
+Configuration
+-------------
+
+This plugin modifies `MFE_CONFIG["PARAGON_THEME_URLS"]` to dynamically load theme CSS files. The default configuration points to:
+
+.. code-block:: json
+
+    MFE_CONFIG["PARAGON_THEME_URLS"] = {
+        "core": {"url": "https://cdn.jsdelivr.net/npm/@edunext/modern-theming-alpha@1.0.0/dist/core.min.css"},
+        "defaults": {"light": "light"},
+        "variants": {
+            "light": {"url": "https://cdn.jsdelivr.net/npm/@edunext/modern-theming-alpha@1.0.0/dist/light.min.css"}
+        }
+    }
+
+
+Usage
+-----
+
+Enable Legacy Theme
+^^^^^^^^^^^^^^^^^^^
+
+Run the following command to enable the legacy theme:
+
+.. code-block:: bash
+
+    tutor enable-legacy-theme
+
+Build and Start
+^^^^^^^^^^^^^^^
+
+After enabling the plugin, rebuild and restart your Tutor environment:
+
+.. code-block:: bash
+
+    tutor config save
+
+    tutor images build mfe
+
+    tutor local start -d
+
+Modern Theming
+--------------
+
+This plugin integrates the Modern Theming system, which is designed to provide a unified look across Open edX's MFEs and legacy pages.
+The theme, called modern-theming, is hosted at `eduNEXT/modern-theming <https://github.com/eduNEXT/modern-theming/tree/main>`_ and leverages Paragon and CSS Variables for runtime styling customization.
+
+Theming includes:
+-----------------
+
+- Support for Paragon UI components with theme-specific styles.
+
+- Comprehensive CSS Variables that allow runtime adjustments without recompiling styles.
+
+- Consistency between MFE-based and legacy Open edX pages.
+
+Prerequisite of the MFE
+-----------------------
+
+The recommended minimum MFE versions should be at least those of SUMAC, preferably using the ednx-release/sumac.master branch.
```

Comment:
Why do we need ednx-release ?

Reply:
For now, because they're the ones that have CSS Vars support at runtime. The community ones don't.

Reply:
I think it would be a good idea to clarify this, so it's clear why we're using our custom versions specifically. I think we can add something like:

```suggestion
This plugin is supported and tested with Open edX releases >= Sumac.

Due to the fact that upstream MFEs in the Sumac release do not support CSS variables at runtime, it is necessary to use our custom MFE branches: `ednx-release/sumac.master`.

If the Open edX release being used already includes runtime support for CSS variables, then using the custom MFEs is not required.
```